src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.tsx

```diff
@@ -29,22 +30,38 @@ const DefaultTranscriptListItem = (props: Props) => {
   const transcriptWidth = scale(transcriptEnd - transcriptStart); // FIXME  this too should be based on relative coordinates of transcript
   const style = { transform: `translateX(${transcriptStartX}px)` };
 
+  const [shouldShowInfo, setShouldShowInfo] = useState(false);
+  const toggleListItemInfo = () => setShouldShowInfo(!shouldShowInfo);
+
   return (
-    <div
-      className={`${styles.defaultTranscriptListItem} ${transcriptsListStyles.row}`}
-    >
-      <div className={transcriptsListStyles.left}>Left</div>
-      <div className={transcriptsListStyles.middle}>
-        <div style={style}>
-          <UnsplicedTranscript
-            transcript={props.transcript}
-            width={transcriptWidth}
-            standalone={true}
-          />
+    <>
+      <div
+        className={`${styles.defaultTranscriptListItem} ${transcriptsListStyles.row}`}
+      >
+        <div className={transcriptsListStyles.left}>Left</div>
+        <div
+          className={transcriptsListStyles.middle}
+          onClick={toggleListItemInfo}
+        >
+          <div style={style}>
+            <UnsplicedTranscript
+              transcript={props.transcript}
+              width={transcriptWidth}
+              standalone={true}
+            />
+          </div>
+        </div>
+        <div
+          className={transcriptsListStyles.right}
+          onClick={toggleListItemInfo}
+        >
+          Right
         </div>
       </div>
-      <div className={transcriptsListStyles.right}>Right</div>
-    </div>
+      {shouldShowInfo ? (
+        <TranscriptsListItemInfo transcriptId={props.transcript.id} />
+      ) : null}
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/77489615-91b0f380-6e30-11ea-9437-b59472eb826d.png)

![image](https://user-images.githubusercontent.com/6834224/77489633-9f667900-6e30-11ea-81af-0f918b459bfb.png)

![image](https://user-images.githubusercontent.com/6834224/77489548-53b3cf80-6e30-11ea-8039-fa172fafae98.png)

I thought the info thing would be the child of DefaultTranscriptListItem, not its sibling

Reply:
Sorry, I got that one wrong yesterday. I made it as its sibling since it can reuse the same grid styling of the default transcript list row.

Reply:
I am not sure what you mean about the grid.

If instead of having a single `row` element, you put its left part in the `left__DefaultTranscriptList` div, the right part in the `right__DefaultTranscriptList` div, and the actual panel in the `middle__DefaultTranscriptList` div, then the grid from the parent component should still work.

Reply:
But shouldn't this be more like an accordion? If we put these columns into the same divs of `DefaultTranscriptListItem` then we need to do three checks to hide/show on toggle.

Reply:
> If we put these columns into the same divs of DefaultTranscriptListItem then we need to do three checks to hide/show on toggle.

You are right about the three checks, but I pictured an accordion item as working like this:

![image](https://user-images.githubusercontent.com/6834224/77541331-711e8300-6e9c-11ea-8d2b-76fa147c6960.png)

rather than like this:

![image](https://user-images.githubusercontent.com/6834224/77541488-a32fe500-6e9c-11ea-94bd-e1a8a413a505.png)

I am not quite sure whether it matters. Personally, I think I would have gone for the first design. @msmanoj what's your take?

Reply:
Sorry, missed this one. The accordion shared component we currently have works like the first design. To me, keeping the Head separate feels like it would become something like a separate toggle button that shows/hides something.

Reply:
@andrey @msmanoj After some meddling I ended up putting the accordion head and body as two rows into a wrapper `div` as:

```
<div className={styles.defaultTranscriptListItem}>
  <div className={styles.row}>accordion head</div>
  <div className={styles.row}>accordion body</div>
</div>
```

I ended up with this taking the following into account:

1. This solves the problem of having the head and body separate from each other.
2. It can use the same grid styling that we already have for the default transcripts list
3. It is neater than putting the body columns alongside the head columns (as mentioned in [this comment](https://github.com/Ensembl/ensembl-client/pull/273#discussion_r397761859))
4. No need to have three separate checks to toggle the accordion body

Reply:
✔️